tests/20-packet-parsing/packet-injector.sh

```diff
@@ -42,12 +53,14 @@ done
 
 echo "Closing native node"
 
-if [ $((TIMEDOUT + FAILED)) -gt 0 ]; then
+if [ -n "$DEBUG" ] || [ $((TIMEDOUT + FAILED)) -gt 0 ]; then
```

Comment:
The build system already uses `DEBUG` for building binaries with debug info.

The build system is not as verbose as it used to be, just remove the redirections (but do that in a separate commit). This will give us accurate time stamps in the CI tests as a bonus. Users who don't want the output can either redirect it to /dev/null themselves, or build with `QUIET=1`.

Reply:
I removed the DEBUG environment variable.